wiki/osu!_tournament_client/osu!tourney/Troubleshooting/de.md

```diff
@@ -0,0 +1,55 @@
+# Problemlösung bei osu!tourney
+
+## Wie erstelle ich eine frische osu!-Installation ohne Deinstallation des aktuellen Spiels?
+
+Kopiere die Datei `osu!.exe` in einen leeren Ordner und führe sie aus.
+
+## Meine osu!-Fenster sind nicht richtig ausgerichtet!
+
+Deaktiviere jegliche sekundären Bildschirme! osu!tourney läuft nur auf dem primären Monitor. Stelle sicher, dass der primäre Bildschirm eine größere Auflösung als der Client hat (standardmäßig 1280 x 720).
+
+## osu!tourney öffnet sich nicht, sondern gibt eine Fehlermeldung aus und/oder schließt sich!
+
+- Überprüfe, dass du in osu! eingeloggt bist und dass die Checkboxen `Benutzername speichern` und `Passwort speichern` gesetzt sind.
+- Versichere dich, dass der Schlüssel `privateserver` in `tournament.cfg` keinen Wert enthält.
+
+## Meine osu!-Songdatenbank ist beim Start beschädigt!
+
+Benutze eine frische osu!-Installation.
+
+## Der Hintergrund wird bei osu!tourney nicht angezeigt!
+
+Stelle sicher, dass die Hintergrunddatei ordnungsgemäß in `/osu!/Skins/User/tournament/background.png` vorhanden ist.
+Siehe den [Skinning-Ratgeber](/wiki/osu!tourney/Skinning) für mehr Details.
+
+## Wie kann ich osu! wieder als normalen Client benutzen?
+
+Ändere den Namen oder lösche die Datei `tournament.cfg`.
+
+## Mein Client ist kein Zuschauer!
+
+Klicke auf den Raumnamen auf dem unteren schwarzen Panel, sodass er **fett gedruckt** erscheint.
+Wenn der Client immer noch nicht zuschaut, klicke auf den Button `Panic`.
+
+## Die Räume tauchen nicht auf!
+
+Es kann passieren, dass dein Raum inkorrekt benannt wurde oder du das falsche Akronym in der Datei `tournament.cfg` benutzt hast.
```

Comment:
```suggestion
Es ist möglich, dass dein Raum inkorrekt benannt wurde oder du das falsche Akronym in der Datei `tournament.cfg` benutzt hast.
```

to me, `kann passieren` signifies something happening outside of the readers control, whereas incorrect naming is very much human error.